### PR TITLE
Doc how to sink to MongoDB

### LINKED ISSUE #2516: Document: feat(sink): Support Mongodb sink
### Related code PR

https://github.com/risingwavelabs/risingwave/pull/17102

### Which part(s) of the docs might be affected or should be updated? And how?

Supports MongoDB sink connector

### Reference

_No response_

### PR DESCRIPTION
**Pull request by WanYixian**
## Description

Doc how to sink to MongoDB, @StrikeW @xxhZs pls help check the accuracy of the words, and let me know if any comments, thanks!

TODO: upgrade public review list and integration summary

## Related code PR

https://github.com/risingwavelabs/risingwave/pull/17102

## Related doc issue

Resolve https://github.com/risingwavelabs/risingwave-docs/issues/2516

## Rendered preview

[Sink data from RisingWave to MongoDB](https://pr-2528.d2fbku9n2b6wde.amplifyapp.com/docs/next/sink-to-mongodb/)

## Checklist

- [ ] I have checked the doc site preview, and the updated parts look good.
- [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`emile-00`, `hengm3467`, & `WanYixian`).

### CODE REVIEW

File: docs/guides/sink-to-mongodb.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+   connector_parameter = 'value', ...
+);
+```
+
+## Parameters
+
+| **Parameter Name**              | **Description**    |
+|--------------------------------|------------------|
+| `mongodb.url`                   | The URL of MongoDB.  |
+| `r#type`                        | Type identifier.  |
```
Comment (StrikeW) [inline docs/guides/sink-to-mongodb.md]: `type`

**Review by StrikeW — APPROVED** [review]
LGTM

File: docs/guides/sink-to-mongodb.md [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+WITH (
+    connector='mongodb',
+    type = 'upsert',
+    mongodb.url = 'mongodb://mongodb:27017/?replicaSet=rs0',
+    collection.name = 'demo.t2',
+    primary_key='id'
+);
+```
+
+Assuming the schema of `t2` is:
```
Comment (ShanlanLi) [inline docs/guides/sink-to-mongodb.md:97]:
```suggestion
Assuming the schema of `t2` is:

```

File: docs/guides/sink-to-mongodb.md [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+| name  | type | pk  |
+|-------|------|-----|
+| id    | int  | ✔   |
+| value | text |     |
+
+Given the record:
```
Comment (ShanlanLi) [inline docs/guides/sink-to-mongodb.md:104]:
```suggestion
Given the record:

```

File: docs/guides/sink-to-mongodb.md (hunk 2 above)
Comment (ShanlanLi) [reply docs/guides/sink-to-mongodb.md:97]: image
The table format is wrong.

**Review by ShanlanLi — APPROVED** [review]
Rest LGTM

File: docs/guides/sink-to-mongodb.md [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+   connector_parameter = 'value', ...
+);
+```
+
+## Parameters
+
+| **Parameter Name**              | **Description**    |
+|--------------------------------|------------------|
+| `mongodb.url`                   | The URL of MongoDB.  |
+| `type`                        | Type identifier.  |
```
Comment (StrikeW) [inline docs/guides/sink-to-mongodb.md]: The `type` defines the type of the sink, we supported `append-only` and `upsert`

**Pull request merged**